pkg/api/v1alpha1/cluster_types.go

```diff
@@ -41,6 +41,11 @@ const (
 	// cluster object.
 	managementAnnotation = "anywhere.eks.amazonaws.com/managed-by"
 
+	// managementComponentsVersionAnnotation is an annotation applied to an EKS-A management cluster pointing to the current version of the management components.
+	// The value for this annotation is expected to correspond to an EKSARelease object version, following semantic version convention: e.g. v0.18.3
+	// This is an internal EKS-A managed annotation, not meant to be updated manually.
+	managementComponentsVersionAnnotation = "anywhere.eks.amazonaws.com/eksa-management-components-version"
```

Comment:
i think there was a comment about dropping eksa?
```suggestion
	managementComponentsVersionAnnotation = "anywhere.eks.amazonaws.com/management-components-version"
	
no strong opinion here just making sure we had an agreement there
```

Reply:
Ah yes, agreed. It's redundant

Reply:
Oh, @d8660091 had mentioned something similar as well, but I thought he was mainly referring to the variable name. Anyway, I've dropped the `eksa` here, which follows convention followed by most of the other annotations as well.